PWM_NativePin.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
-class PWM_NativePin {
+#include "PWM_Pins.h"
+
+class PWM_NativePin : public PWM_Pins {
```

Comment:
Indicates that this is like the general `PWM_Pins` type of thing (which you'll see below, alphabetic order...).